docs/feature/connectivity/index.md

```diff
@@ -1,85 +1,608 @@
-- [](#integrate-metrics)
-- [](#performance)
+In order to provide a CrateDB instance for testing purposes, use, for
+example, Docker.
+```shell
+docker run --rm -it --publish=4200:4200 --publish=5432:5432 crate/crate:nightly
+```
+
+
+::::::{tab-set}
+
+:::::{tab-item} C# / Npgsql
+For more information see [Npgsql] documentation.
+
+`program.csproj`
+```cs
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <PackageReference Include="Npgsql" Version="8.0.5" />
+  </ItemGroup>
+</Project>
+```
+`program.cs`
+```csharp
+using Npgsql;
+using NpgsqlTypes;
+
+var connString =
+    $"Host=localhost;Port=5432;SSL Mode=Disable;" +
+    $"Username=crate;Password=;Database=testdrive";
+
+using (var conn = new NpgsqlConnection(connString)) {
+    conn.Open();
+
+    var mountains = new List<string>();
+    using (var cmd = new NpgsqlCommand("SELECT mountain FROM sys.summits ORDER BY 1 LIMIT 25", conn))
+    using (var reader = cmd.ExecuteReader()) {
+        while (await reader.ReadAsync()) {
+            mountains.Add(reader.GetString(0));
+        }
+        Console.WriteLine($"Mountains: {string.Join(",", mountains)}");
+    }
+    conn.Close();
+}
+```
+:::::
+
+:::::{tab-item} Golang / pgx
+For more information see [pgx][golang-pgx] documentation.
+```shell
+go mod add pgx
+```
+```golang
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/jackc/pgx/v5"
+)
+
+func main() {
+	dbUrl := "postgres://crate:@localhost:5432/testdrive"
+	conn, err := pgx.Connect(context.Background(), dbUrl)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
+		os.Exit(1)
+	}
+	defer conn.Close(context.Background())
+
+	var mountain string
+	err = conn.QueryRow(context.Background(), "SELECT mountain FROM sys.summits ORDER BY 1 LIMIT 25").Scan(&mountain)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "QueryRow failed: %v\n", err)
+		os.Exit(1)
+	}
+
+	fmt.Println(mountain)
+}
+```
+:::::
+
+:::::{tab-item} Java / JDBC
+JDBC is a standard Java API that provides a common interfaces for accessing
+databases in Java.
+For more information see [pgJDBC] documentation.
+```xml
+<project>
+    <dependencies>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.7.4</version>
+        </dependency>
+        <dependency>
+            <groupId>io.crate</groupId>
+            <artifactId>crate-jdbc</artifactId>
+            <version>2.7.0</version>
+        </dependency>
+    </dependencies>
+</project>
+```
+```java
+import java.sql.*;
+import java.util.Properties;
+
+public class Main {
+    public static void main(String[] args) {
+        try {
+            Properties properties = new Properties();
+            properties.put("user", "crate");
+            properties.put("password", "");
+            properties.put("ssl", false);
+            Connection conn = DriverManager.getConnection(
+                "jdbc:crate://localhost:5432/",
```

Comment:
please consider to add a comment here about the two different drivers

Reply:
Thanks. Do you agree with that variant?

https://github.com/crate/cratedb-guide/blob/525483fa894f1da95bcffce5e1aa91fea30b6be5/docs/feature/connectivity/index.md?plain=1#L183-L197

- => See: GH-138. Please respond there to further improve that section. For example, we could add inline comments or a link to the relevant documentation page, when applicable.